From 99105e765c31b598746b849e66e7a9106dcefa24 Mon Sep 17 00:00:00 2001
From: Maarten Lankhorst <maarten.lankhorst@canonical.com>
Date: Wed, 27 Mar 2013 13:06:43 +0000
Subject: makefiles: Add missing headers.

I even compile time tested this on a panda with make dist!

Signed-off-by: Maarten Lankhorst <maarten.lankhorst@canonical.com>
---
diff --git a/exynos/Makefile.am b/exynos/Makefile.am
index 539aea0..0a2663a 100644
--- a/exynos/Makefile.am
+++ b/exynos/Makefile.am
@@ -10,7 +10,11 @@ libdrm_exynos_ladir = $(libdir)
 libdrm_exynos_la_LDFLAGS = -version-number 1:0:0 -no-undefined
 libdrm_exynos_la_LIBADD = ../libdrm.la @PTHREADSTUBS_LIBS@
 
-libdrm_exynos_la_SOURCES = exynos_drm.c exynos_fimg2d.c
+libdrm_exynos_la_SOURCES = \
+	exynos_drm.c \
+	exynos_fimg2d.c \
+	fimg2d.h \
+	fimg2d_reg.h
 
 libdrm_exynoscommonincludedir = ${includedir}/exynos
 libdrm_exynoscommoninclude_HEADERS = exynos_drm.h
diff --git a/freedreno/Makefile.am b/freedreno/Makefile.am
index f46a780..ba9bd68 100644
--- a/freedreno/Makefile.am
+++ b/freedreno/Makefile.am
@@ -10,7 +10,15 @@ libdrm_freedreno_ladir = $(libdir)
 libdrm_freedreno_la_LDFLAGS = -version-number 1:0:0 -no-undefined
 libdrm_freedreno_la_LIBADD = ../libdrm.la @PTHREADSTUBS_LIBS@
 
-libdrm_freedreno_la_SOURCES = freedreno_device.c freedreno_pipe.c freedreno_ringbuffer.c freedreno_bo.c
+libdrm_freedreno_la_SOURCES = \
+	freedreno_device.c \
+	freedreno_pipe.c \
+	freedreno_priv.h \
+	freedreno_ringbuffer.c \
+	freedreno_bo.c \
+	kgsl_drm.h \
+	list.h \
+	msm_kgsl.h
 
 libdrm_freedrenocommonincludedir = ${includedir}/freedreno
 libdrm_freedrenocommoninclude_HEADERS = freedreno_drmif.h freedreno_ringbuffer.h
--
cgit v0.9.0.2-2-gbebe
